notebooks/Hugging-Face-BERT.ipynb

```diff
@@ -0,0 +1,972 @@
+    "outputs = traced_mlm_model(encoded_inputs['input_ids'], encoded_inputs['token_type_ids'], encoded_inputs['attention_mask'])\n",
+    "most_likely_token_ids = [torch.argmax(outputs[0][i, pos, :]) for i, pos in enumerate(pos_masks)]\n",
+    "unmasked_tokens = enc.decode(most_likely_token_ids).split(' ')\n",
+    "unmasked_sentences = [masked_sentences[i].replace('[MASK]', token) for i, token in enumerate(unmasked_tokens)]\n",
+    "for sentence in unmasked_sentences:\n",
+    "    print(sentence)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f3b5c2f",
+   "metadata": {},
+   "source": [
+    "<a id=\"4\"></a>\n",
+    "## 4. Compiling with Torch-TensorRT"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "3fbb2ac5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: [Torch-TensorRT] - For input input_ids, found user specified input dtype as Int32, however when inspecting the graph, the input type expected was inferred to be Float\n",
+      "The compiler is going to use the user setting Int32\n",
+      "This conflict may cause an error at runtime due to partial compilation being enabled and therefore\n",
+      "compatibility with PyTorch's data type convention is required.\n",
+      "If you do indeed see errors at runtime either:\n",
+      "- Remove the dtype spec for input_ids\n",
+      "- Disable partial compilation by setting require_full_compilation to True\n",
+      "WARNING: [Torch-TensorRT] - For input attention_mask.1, found user specified input dtype as Int32, however when inspecting the graph, the input type expected was inferred to be Double\n",
+      "The compiler is going to use the user setting Int32\n",
+      "This conflict may cause an error at runtime due to partial compilation being enabled and therefore\n",
+      "compatibility with PyTorch's data type convention is required.\n",
+      "If you do indeed see errors at runtime either:\n",
+      "- Remove the dtype spec for attention_mask.1\n",
+      "- Disable partial compilation by setting require_full_compilation to True\n",
+      "WARNING: [Torch-TensorRT] - For input token_type_ids, found user specified input dtype as Int32, however when inspecting the graph, the input type expected was inferred to be Float\n",
+      "The compiler is going to use the user setting Int32\n",
+      "This conflict may cause an error at runtime due to partial compilation being enabled and therefore\n",
+      "compatibility with PyTorch's data type convention is required.\n",
+      "If you do indeed see errors at runtime either:\n",
+      "- Remove the dtype spec for token_type_ids\n",
+      "- Disable partial compilation by setting require_full_compilation to True\n",
+      "WARNING: [Torch-TensorRT TorchScript Conversion Context] - Tensor DataType is determined at build time for tensors not marked as input or output.\n",
```

Comment:
Do you think its worth setting the logging level to Error so that these warnings don't get dumped?

Reply:
Sure, that sounds fine. How do I change the logging level?